packages/nextra-theme-docs/src/mdx-components.tsx

```diff
@@ -216,7 +206,14 @@ const A = ({ href = '', ...props }) => (
   />
 )
 
-export const getComponents = () => {
+export const getComponents = (
+  isRawLayout = false,
+  components?: DocsThemeConfig['components']
+): DocsThemeConfig['components'] => {
```

Comment:
Can we make this an object?

```suggestion
export const getComponents = ({
  isRawLayout = false,
  components?: DocsThemeConfig['components']
}): DocsThemeConfig['components'] => {
```

Reply:
Sure!

Reply:
@shuding done